scripts/components/Repo.js

```diff
@@ -1,19 +1,24 @@
 'use strict';
 
 import React, { PropTypes } from 'react';
-import PureRenderMixin from 'react/lib/ReactComponentWithPureRenderMixin';
+import shallowEqual from 'react/lib/shallowEqual';
 import { Link } from 'react-router';
 
-const Repo = React.createClass({
-  mixins: [PureRenderMixin],
+class Repo extends React.Component {
 
-  propTypes: {
-    repo: PropTypes.object.isRequired,
-    owner: PropTypes.object.isRequired
-  },
+  // TODO: make it either a HigherOrderComponent
```

Comment:
So if I understand correctly ES7 class properties, you mean something like this?

```js
import shouldComponentUpdatePure from './utils/shouldComponentUpdatePure'

class Foo extends React.Component {
  shouldComponentUpdate = shouldComponentUpdatePure

  render () {
    return <div>foo</div>
  }
}
```